docs/docs-graphql/schema.graphqls

```diff
@@ -1435,6 +1440,9 @@ type Function {
     """A doc string for the argument, if any"""
     description: String = ""
 
+    """Patterns to ignore when loading the contextual argument value."""
+    ignore: [String!] = []
```

Comment:
Can we call this something like `defaultPathIgnore`? Something to make it clear that this applies *only* to the `defaultPath`?

Alternatively, we could add a new method `withPathArg`, for clarity, but that might be less popular.

Reply:
The thing is that it's not only for the defaulth path though. It can be used as a replacement for _views_ for example, applying to a local directory referenced from the CLI.